content/reference/rest-api/engine-diagrams/create.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Engine Diagrams: create'
+weight: 10
+
+menu:
+  main:
+    name: 'create'
+    identifier: 'rest-api-engine-diagrams-create'
+    parent: 'rest-api-engine-diagrams'
+    pre: 'Creates or updates an engine diagram.'
+---
+
+Creates a new engine diagram if it doesn't exist yet or updates an engine diagram with a new version.
+
+# Request
+
+## HTTP Request
+
+```
+PUT https://cawemo.com/api/v1/engine-diagrams
```

Comment:
Isn't here the project path parameter missing or to which project would the diagram be added?

Reply:
Good catch, it's missing.